h/security/permission_map.py

```diff
@@ -66,7 +66,10 @@
         [p.group_has_user_as_admin],
         [p.group_has_user_as_moderator],
     ],
-    Permission.Group.MEMBER_ADD: [[p.group_matches_authenticated_client_authority]],
+    Permission.Group.MEMBER_ADD: [
+        [p.group_member_add],
+        [p.group_matches_authenticated_client_authority],
+    ],
```

Comment:
So the `MEMBER_ADD` permission will be granted if _either_ `group_member_add()` returns `True` _or_ `group_matches_authenticated_client_authority()` returns `True`.